erpnext/controllers/stock_controller.py

```diff
@@ -1655,14 +1657,15 @@ def future_sle_exists(args, sl_entries=None, allow_force_reposting=True):
 
 	or_conditions = get_conditions_to_validate_future_sle(sl_entries)
 
+	args["posting_datetime"] = datetime.strptime(f"{args['posting_date']} {args['posting_time']}", "%Y-%m-%d %H:%M:%S")
```

Comment:
use the [get_combine_datetime](https://github.com/frappe/erpnext/blob/develop/erpnext/stock/utils.py#L654) function